package.json

```diff
@@ -46,10 +46,15 @@
     "micromatch": "^4.0.8",
     "p-map": "^4.0.0",
     "semver": "^7.6.0",
-    "snyk-config": "^5.0.0",
+    "snyk-config": "^5.2.0",
     "tslib": "^1.9.3",
     "uuid": "^8.3.0"
   },
+  "overrides": {
+    "@yarnpkg/core" : {
+      "cross-spawn": "^7.0.5"
+    }
+  },
```

Comment:
Also the types should match v18 rather than 16

Reply:
leaving types at 16 since the yarnpkg version doesn't work with types 18. Will resolve that in later PR.